extra/write_throughput_benchmark.rst

```diff
@@ -0,0 +1,135 @@
+
+Before we start, we need to tell pgbench to initialize the benchmarking environment by creating test tables. Then, we need to connect to the Citus coordinator node and distribute the table that we're going to run INSERT benchmarks on.
+
+To initialize the test environment and distribute the related table, you need to get a connection string to the cluster. You can get this connection string from your Citus Cloud dashboard. Then, you need to run the following commands::
+
+  pgbench -i connection_string_to_coordinator
+  psql connection_string_to_coordinator
+  
+  psql=> SELECT create_distributed_table('pgbench_history', 'aid');
+
+
+Create SQL File for pgbench
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pgbench runs the given SQL commands repeatedly and reports results. For this benchmark run, we will use the INSERT command that comes with pgbench.
+
+To create the related SQL commands, create a file named insert.sql and paste the following lines into it::
+
+  \set nbranches :scale
+  \set ntellers 10 * :scale
+  \set naccounts 100000 * :scale
+  \setrandom aid 1 :naccounts
+  \setrandom bid 1 :nbranches
+  \setrandom tid 1 :ntellers
+  \setrandom delta -5000 5000
+  INSERT INTO pgbench_history (tid, bid, aid, delta, mtime) VALUES (:tid, :bid, :aid, :delta, CURRENT_TIMESTAMP);
+
+
+Benchmark INSERT commands
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, pgbench opens a single connection to the database and sends INSERT commands through this connection. To benchmark write throughput, we're going to open parallel connections to the database and issue concurrent commands. In particular, we're going to use pgbench's -j parameter to specify the number of concurrent threads and -c parameter to specify the number of concurrent connections. We will also set the duration for our tests to 30 seconds using the -T parameter.
+
+To run pgbench with these parameters, simply type::
+
+  pgbench connection_string_to_coordinator -j 64 -c 256 -f insert.sql -T 30
+
+Please note that these parameters open 256 concurrent connections to Citus. If you're running Citus on your own instances, you will need to increase the default max_connections setting.
+
+.. _citus_update_throughput_benchmark:
+
+Benchmark UPDATE Throughput
+---------------------------
+
+Initialize and Distribute Tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before we start, we need to tell pgbench to initialize the benchmarking environment by creating test tables. Then, we need to connect to the Citus coordinator node and distribute the table that we're going to run UPDATE benchmarks on.
+
+To initialize the test environment and distribute the related table, you need to get a connection string to the cluster. You can get this connection string from your Citus Cloud dashboard. Then, you need to run the following commands::
+
+  pgbench -i connection_string_to_coordinator
+  psql connection_string_to_coordinator
+  
+  psql=> /* INSERT and UPDATE tests run on different distributed tables */
```

Comment:
Should we remove "psql=>" from the comment line and change comments to bash type comments ('#' instead of '/* */')